tests/test_pretty.py

```diff
@@ -205,6 +205,15 @@ def test_pretty_string():
     assert pretty_string(test) == expected
 
 
+def test_pretty_file():
```

Comment:
You could use the `tmp_path` fixture in `pytest` (see https://docs.pytest.org/en/6.2.x/tmpdir.html)